backend/eventloop/workspace_resource_event_loop_test.go

```diff
@@ -21,7 +21,7 @@ var _ = Describe("isManagedEnvironmentConnectionUserError function Test", func()
 			rootErr := fmt.Errorf("Get \"https://api.ireland.burr-on-aws.com:6443/api?timeout=32s\": dial tcp: lookup api.ireland.burr-on-aws.com on 172.30.0.10:53: no such host")
 			err := fmt.Errorf("%s: %w", shared_resource_loop.UnableToCreateRestConfigError, rootErr)
 
-			result := isManagedEnvironmentConnectionUserError(err, log)
+			result := application_event_loop.IsManagedEnvironmentConnectionUserError(err, log)
```

Comment:
Should we move the tests to the same package as the function? It's not significant but makes it easier to find it. Everything else looks great.

Reply:
Ah, yes I forgot to move unit tests after I moved the function definition.